### DISCUSSION TITLE
Ability to use custom LLM

### DISCUSSION THREAD

**Discussion by Rostezkiy** [pieces-app/support#119] [Ideas]
Hello! 
I think it would be a nice addition, if i could use custom LLM in Pieces, not only mistral/meta and others provided by default. 
For example, i have already downloaded model; maybe there should be a field with path to model that is using or something like that. 
Encountered that while i had some internet problems, so my download didn't continue later and just stuck on "Initialization". 
Thanks!

**Comment by rosie-at-pieces** [comment]
We've gotten this request a fair amount lately; unfortunately this would require a fairly large shift on our end and it's not likely we're going to be able to get to this for a while. But we do love the idea and will keep thinking on the best way to get to it!

**Comment by suryapratap** [comment]
Considering that pieces-os has moved to an external ollama install, is it possible to configure custom models that are available via ollama.

**Comment by onkanat** [comment]
Pieces' Ollama works fine.
I installed curl with qwen2.5-coder etc.
and I use it with another chat user interface.
How do the models I installed in Pieces Desktop work?
